airbyte-webapp/src/views/layout/SideBar/components/SidebarDropdownMenu.tsx

```diff
@@ -0,0 +1,29 @@
+import { Menu } from "@headlessui/react";
+import classNames from "classnames";
+import React from "react";
+
+import styles from "./SidebarDropdownMenu.module.scss";
+
+const SidebarDropdownMenu: React.FC<{
+  children: React.ReactNode;
+  options?: React.ReactNode[];
+}> = ({ children, options }) => {
+  return (
+    <Menu className={styles.sidebarMenu} as="div">
+      {({ open }) => (
+        <>
+          <Menu.Button className={classNames(styles.button, { [styles.open]: open })}>{children}</Menu.Button>
+          <Menu.Items className={styles.items} as="div">
```

Comment:
We should use the `active` property of an item here to actually highlight the focused item properly. At the moment it doesn't seem to highlight any item in the menu when navigating the menu (via keyboard or mouse).

Reply:
Added support for keyboard interaction